v1/api/auth.ts

```diff
@@ -51,6 +51,6 @@ export const authRoutes = (store: StoreI) => async (server: FastifyTypebox): Pro
   }, async (request, reply) => {
     const { hash } = request.params
     await store.revocations.revoke(hash)
-    return reply.code(200).send()
+    return await reply.code(200).send()
```

Comment:
I don't think the await is necessary when returning from the async function. I think standard.js usually complains when you do this.

If this is to satisfy your linter then feel free to just follow that.

Reply:
yeah I actually took the `await` out but the linter insisted on adding it back in LOL

Reply:
Oh okay. Well whatever the linter wants is fine. Doesn't make much of a difference